beacon-chain/chaintest/backend/simulated_backend.go

```diff
@@ -176,17 +176,29 @@ func (sb *SimulatedBackend) RunForkChoiceTest(testCase *ForkChoiceTestCase) erro
 // algorithm, then compare the output with the expected output from the YAML file.
 func (sb *SimulatedBackend) RunShuffleTest(testCase *ShuffleTestCase) error {
 	defer db.TeardownDB(sb.beaconDB)
-	seed := common.BytesToHash([]byte(testCase.Seed))
-	indexList := make([]uint64, len(testCase.Input.Validators), len(testCase.Input.Validators))
-	for i, v := range testCase.Input.Validators {
-		indexList[i] = uint64(v.OriginalIndex)
+	seed := common.HexToHash(testCase.Seed)
+	testIndices := make([]uint64, testCase.Count, testCase.Count)
+	for i := uint64(0); i < testCase.Count; i++ {
+		testIndices[i] = i
 	}
-	output, err := utils.ShuffleIndices(seed, indexList)
+	shuffledList := make([]uint64, testCase.Count)
+	for i := uint64(0); i < testCase.Count; i++ {
+		si, err := utils.UnShuffledIndex(i, testCase.Count, seed)
```

Comment:
why is this unshuffled here ?
```suggestion
		si, err := utils.ShuffledIndex(i, testCase.Count, seed)
		if err != nil {
			log.Error(err)
			return err
		}
		shuffledList[i] = si
```

Reply:
<img width="609" alt="Screen Shot 2019-05-26 at 10 22 56 AM" src="https://user-images.githubusercontent.com/5848741/58377517-53dd0780-7fa0-11e9-931e-2dd3347954c9.png">